projects/storefrontapp-e2e-cypress/cypress/helpers/big-happy-path.ts

```diff
@@ -0,0 +1,124 @@
+import { user, cart, product } from '../sample-data/big-happy-path';
+import { login, register } from './auth-forms';
+import { fillPaymentDetails, fillShippingAddress } from './checkout-forms';
+
+export function signOut() {
+  cy.selectUserMenuOption('Sign Out');
+}
+
+export function signInAndRegister() {
+  cy.getByText(/Sign in \/ Register/i).click();
+  cy.getByText('Register').click();
+  register(user);
```

Comment:
maybe move the assertion that user is logged in, in here.
cy.get('.cx-login-status__greet').should('contain', user.fullName);